.travis.yml

```diff
@@ -86,8 +86,8 @@ script:
       export CONFIGURE_OPTIONS="--enable-code-coverage";
     fi
   - ../configure --enable-unit $CONFIGURE_OPTIONS --with-simulatorbin=$(pwd)/../ibmtpm/src/tpm_server
```

Comment:
Missing scan-build prefix her

Reply:
We are not running static analysis on tpm_server.  We cannot change the simulator source and it has many problems already if we even enable gcc/clang warnings.

Reply:
```
If an analyzed project uses an autoconf generated configure script, you will probably need to run configure script through scan-build in order to analyze the project.
```
(source https://clang-analyzer.llvm.org/scan-build.html)